src/cloudWatchLogs/cloudWatchLogsUtils.ts

```diff
@@ -35,6 +63,15 @@ export function parseCloudWatchLogsUri(uri: vscode.Uri): {
         parameters: JSON.parse(uri.query),
     }
 }
+/**
+ * Determines if loadOlder codelense should be visible on virtual document
+ * @param uri CloudWatchLogs Document URI
+ * @returns
+ */
+export function loadOlderRelevant(uri: vscode.Uri): boolean {
```

Comment:
Common JS convention follows this naming format `can${action}` e.g. `canShowPreviousLogs` for these types of functions.